tests/android/MASVS-CODE/MASTG-TEST-0x27-1.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: Testing for URL Loading in WebViews
+ID: MASTG-TEST-0x27-1
+Link: https://mas.owasp.org/MASTG/tests/android/MASVS-CODE/MASTG-TEST-0027/
+Platform: android
+type: [static]
+MASVS v1: ['MSTG-PLATFORM-2']
+MASVS v2: ['MASVS-CODE-4']
+---
+
+## Overview
+
+By default, navigation events inside of a WebView will redirect to the default browser application. However, it is possible to stay within the WebView and handle all new page loads. This can be dangerous, as the new page may be malicious and interact with either the JavaScript bridge, or phish the user. The application should monitor navigation events inside the WebView to make sure that only legitimate pages are loaded, while others are redirected to the browser application.
+
+## Steps
+
+1. Examine the application's code (see @MASTG-TECH-0023)
+2. Look for occurrences of WebViews being used and examine if they are configured with a custom `WebViewClient`.
```

Comment:
```suggestion
2. Look for occurrences of WebViews being used and examine if they are configured with a custom `WebViewClient`. This can be done e.g., by calling `webview.setWebViewClient(new MyWebViewClient());`. Alternatively, you can look for any class extending `WebViewClient`.
```